web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -75,6 +75,17 @@
             :rules="awardNumberRules"
           />
         </div>
+        <div v-if="!embargoed">
+          <v-select
+            v-model="license"
+            :items="dandiLicenses"
+            :label="`License ${embargoed ? '' : '*'}`"
```

Comment:
```suggestion
        <div v-if="!embargoed">
          <v-select
            v-model="license"
            :items="dandiLicenses"
            label="License*"
```

Just realized, the v-if actually implies that this license dropdown won't appear for an embargoed dandiset, so my earlier suggestion doesn't make sense.